apps/tasks/lib/controllers/assignee_select_controller.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/foundation.dart';
+import 'package:helpwave_widget/loading.dart';
+import 'package:tasks/dataclasses/task.dart';
+import 'package:tasks/services/current_ward_svc.dart';
+import 'package:tasks/services/organization_svc.dart';
+import 'package:tasks/services/task_svc.dart';
+import '../dataclasses/user.dart';
+
+/// The Controller for selecting a [User] as the assignee of a [Task]
+class AssigneeSelectController extends ChangeNotifier {
+  /// The [LoadingState] of the Controller
+  LoadingState _state = LoadingState.initializing;
+
+  LoadingState get state => _state;
+
+  set state(LoadingState value) {
+    _state = value;
+    notifyListeners();
+  }
+
+  /// The selected [User] identifier
+  String? _selected;
+
+  String? get selected => _selected;
+
+  User? get selectedUser => _users.firstWhere((user) => user.id == selected);
+
+  /// The currently loaded user
+  List<User> _users = [];
+
+  /// The currently loaded tasks
+  List<User> get users => _users;
+
+  /// The identifier of the current [Task] it determines whether
+  /// changes are pushed to the server
+  String? _taskId;
+
+  AssigneeSelectController({String? selected, String? taskId}) {
+    _selected = selected;
+    _taskId = taskId;
+    load();
+  }
+
+  /// Loads the tasks
+  Future<void> load() async {
+    state = LoadingState.loading;
+    String? currentOrganization = CurrentWardService().currentWard?.organizationId;
+    if(currentOrganization == null){
+      if(kDebugMode){
+        print("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
```

Comment:
I think in the future we should think of using a logger instead of print

https://pub.dev/packages/logger

Reply:
Agreed, I opened #445 for this